### ISSUE TITLE
Map editor request

### ISSUE THREAD

**Issue by Jummit** [nesbox/tic.computer#178]
It would be cool to have some features:
1. bigger patterns (you can only take one sprite at the time jet)
2. multiple maps instead of a big map
3. fill, select and copy tool
4. maybe (but not required) something like this: [http://doc.mapeditor.org/manual/using-the-terrain-tool/](url)

**Comment by darkhog** [comment]
I'd like to get one useful map feature from RPG Maker that kinda fits into "bigger patterns" category. You can in the tile selector click and drag to select a rectangle of tiles and then stamp them on the map. So e.g. if you have a tree that is 2x3 tiles, you can put entire tree at once

**Comment by Jummit** [comment]
Yes, thats what i wanted to say. Maybe not select rectangles only, you could also select specific tiles in a T-shape maybe.

**Comment by darkhog** [comment]
I think this would be too cumbersome to implement. There's a reason that RPG Maker doesn't do it even though in case of some third-party tilesets this feature would be useful.

**Comment by Jummit** [comment]
It is realy annoying to put every single sprite in the map by hand. It is impossible to make a forest when you have to do everything alone. ):

**Comment by darkhog** [comment]
Yep, and that's why rectangle tile selection, like one in RPG Maker should be implemented. Anything above simple rectangle selection is an overkill.
